### PR TITLE
FW-4737 Move part of speech to base dictionary entry model

### PR DESCRIPTION
**Pull request by bronson-c**
### Description of Changes
Moves part of speech to the dictionary entry model rather than the translation model. Tests and serializers are updated accordingly.

### Checklist
- [x] README / documentation has been updated if needed
- [x] PR title / commit messages contain Jira ticket number if applicable
- [x] Tests have been updated / created if applicable
- [x] Admin Panel has been updated if models have been added or modified
- [x] Migrations have been updated and committed if applicable
- [x] Team Postman workspace has been updated if applicable

### Reviewers Should Do The Following:
- [x] Review the code changes here
- [ ] Pull the branch and test locally

### Additional Notes
N/A

### CODE REVIEW

File: firstvoices/backend/serializers/site_data_serializers.py [hunk 1]
```diff
@@ -122,11 +122,10 @@ def get_optional(self, dictionaryentry):
                 ),
                 **(
                     {
-                        "Part of Speech": dictionaryentry.translation_set.first().part_of_speech.title,
+                        "Part of Speech": dictionaryentry.part_of_speech.title,
                     }
                     if dictionaryentry.translation_set.first() is not None
```
Comment (sarahfirstvoices) [inline firstvoices/backend/serializers/site_data_serializers.py:127]: Thanks for updating this serializer. I don't think we need to check the translation_set at all here anymore

**Review by sarahfirstvoices — APPROVED** [review]

**Comment by sarahfirstvoices** [comment]
Approved, assuming the tests pass after fixing the migration numbering

**Review by NolanVH — APPROVED** [review]
The one thing I am wondering about is if the Admin pages need to be updated after these changes. Otherwise, all changes look good to me!

**Comment by bronson-c** [comment]
> The one thing I am wondering about is if the Admin pages need to be updated after these changes. Otherwise, all changes look good to me!

Good catch, the admin site is broken. Will fix in the next commit

**Pull request merged**